internal/tasks/manifests.go

```diff
@@ -794,3 +794,82 @@ func (j *Janitor) buildClairManifest(account keppel.Account, manifest keppel.Man
 
 	return result, nil
 }
+
+var getDigestForIndexStatesToResubmitQuery = sqlext.SimplifyWhitespace(fmt.Sprintf(`
+	SELECT digest from vuln_info
+	WHERE index_finished_at IS NOT NULL
+		AND index_state != $1
+		AND status != '%s'
+	LIMIT $2;
+`, clair.PendingVulnerabilityStatus))
+
+func (j *Janitor) CheckClairManifestState() error {
+	indexStateHash, err := j.cfg.ClairClient.GetIndexStateHash()
+	if err != nil {
+		return err
+	}
+
+	total, err := j.db.SelectInt("SELECT COUNT (*) FROM vuln_info")
+	if err != nil {
+		return err
+	}
+	inPending, err := j.db.SelectInt(fmt.Sprintf("SELECT COUNT(*) FROM vuln_info WHERE status = '%s'", clair.PendingVulnerabilityStatus))
+	if err != nil {
+		return err
+	}
```

Comment:
Sequence scans are expensive, so we should do both at once. Something along those lines:

```go
query := `SELECT COUNT(*), COUNT(CASE WHEN status = 'Pending' THEN TRUE ELSE NULL END) FROM vuln_info`
err := j.db.QueryRow(query).Scan(&total, &inPending)
```